src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -300,3 +313,31 @@ def get_partition_hosts(self, partition_key, host_list):
             return hosts, partiton_name
         else:
             return host_list, None
+
+    def log_path(self, test_name=None):
+        """Determine log path for both servers and clients"""
+        # Determine the path and name of the daos server log using the
+        # D_LOG_FILE env or, if not set, the value used in the doas server yaml
+        if test_name:
+            self.test_id = test_name
+        self.log_dir, self.server_log = os.path.split(
+            os.getenv("D_LOG_FILE", SERVER_LOG))
+        self.client_log = os.path.join(self.log_dir,
+                                       self.test_id + "_" + CLIENT_LOG)
+        # To generate the seperate client log file
+        self.orterun_env = {'D_LOG_FILE':self.client_log}
+
+    def collect_separate_logs(self):
```

Comment:
Would it make more sense to inject the test variant name into the server log name when we start the servers, as opposed to renaming the logs after the run?  This could be done in the `create_server_yaml()` method by updating the `default_value_set["servers"][0]["log_file"]` value after taking into account the test yaml updates.